cumulus_library/study_manifest.py

```diff
@@ -179,11 +206,13 @@ def get_export_table_list(self) -> list[ManifestExport] | None:
 
     def get_all_generators(self) -> list[str]:
         """Convenience method for getting files that generate sql queries"""
-        return (
-            self.get_table_builder_file_list()
-            + self.get_counts_builder_file_list()
-            + self.get_statistics_file_list()
-        )
+        files = self.get_file_list()
+        return [file for file in files if file.endswith(".py")]
```

Comment:
I feel like this is a pattern I'm seeing a bit in this PR - do you want to add an optional suffix argument to `get_file_list`?

Reply:
i could? i kind of like the semantics of a function that is named after what it does, but those could just be wrappers for something in one place that takes an arg